packages/sshnoports/lib/sshnpd/sshnpd_impl.dart

```diff
@@ -309,7 +305,9 @@ class SSHNPDImpl implements SSHNPD {
       }
 
       // Check to see if the ssh Publickey is already in the file if not append to the ~/.ssh/authorized_keys file
-      var authKeys = File('${sshHomeDirectory}authorized_keys');
+      var authKeysFilePath = [homeDirectory, '.ssh', 'authorized_keys']
+          .join(Platform.pathSeparator);
+      var authKeys = File(authKeysFilePath);
```

Comment:
While you're in this code could you add the check to see if it is already in the file? Currently it duplicates entries.

Reply:
It does check, lines 316/314 - 318/316 (just after this diff)